### ISSUE TITLE
Move Ombi v4 releases from Ombi.Releases to the main repository

### ISSUE THREAD

**Issue by douglasparker** [tidusjar/Ombi#3807]
**Describe the bug**
It is extremely hard to follow the progress of Ombi V4 because releases are made in the Ombi.Releases repository. The release notes do not actually link to the issues being fixed here in the main repository, which means I have to manually lookup each issue that was resolved to know what was fixed between releases.

Please use one central repository for Ombi or provide manual release notes for everyone's convenience.

**To Reproduce**
N/A

**Expected behavior**
N/A

**Screenshots**
N/A

**Logs (Logs directory where Ombi is located)**
N/A

**Desktop (please complete the following information):**
 N/A

**Ombi Version (please complete the following information):**
 N/A

**Additional context**
N/A

**Comment by tidusjar** [comment]
I will be doing this as soon as v4 has been merged in with develop
